### ISSUE TITLE
Let bin/fletch-vm fail in a friendlier way when passed a .dart file

### ISSUE THREAD

**Issue by kasperl** [dart-lang/fletch#181]
Current output:

```
bin/fletch-vm ../sdk-300915/hello.dart ../../src/vm/main.cc:147: error: File is not a snapshot.
Aborted (core dumped)
```

**Comment by kasperl** [comment]
Fix under review: https://codereview.chromium.org/1377463004/.

**Comment by kasperl** [comment]
Fix landed in b08a600cfe24a4b115fe957baa7d91de4b32edb4.
